modules/test/playwright/pages/multi-factor-authentication/MultiFactorAuthenticationConfigurationPage.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {InstanceSettingsPage} from '../configuration-admin-web/InstanceSettingsPage';
+
+export class MultiFactorAuthenticationConfigurationPage {
```

Comment:
Please sort Locator declarations, constructor variables, and class functions by name